src/aap_eda/api/views/project.py

```diff
@@ -229,11 +229,20 @@ def partial_update(self, request, pk):
                 "description", project.description
             )
             project.save()
-        except IntegrityError as e:
-            return Response(
-                {"errors": str(e)},
-                status=status.HTTP_409_CONFLICT,
-            )
+        except IntegrityError:
+            project_name = serializer.validated_data["name"]
+            if not project_name:
+                raise api_exc.Conflict(
+                    detail="Project name cannot be an empty string."
+                )
+            project_name_exists = models.Project.objects.filter(
+                name=project_name
+            ).exists()
+            if project_name_exists:
+                raise api_exc.Conflict(
```

Comment:
@Dostonbek1 Wouldn't this be taken care of by this line https://github.com/ansible/eda-server/blob/f7b2c73120dae4344288e74786d842d4049c6bc4/src/aap_eda/core/models/project.py#L51

Reply:
@mkanoor This seems to work only on POST endpoint:
![Screenshot 2023-08-07 at 1 22 36 PM](https://github.com/ansible/eda-server/assets/31990136/cd0dac01-7f1b-4747-9fab-0ff5def1cf73)

But on PATCH, I get the following:
![Screenshot 2023-08-07 at 1 22 45 PM](https://github.com/ansible/eda-server/assets/31990136/44db0a98-fdfe-4291-b0b9-98c48d5fb969)